# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1517257149 18000
#      Mon Jan 29 15:19:09 2018 -0500
# Node ID d88323efe969184445c34ea709d8785fe788e097
# Parent  2ab1154f2c7d333e2157a252edaaa1145f602af7
don't resize variable editor columns on every update (bug #52988)

* variable-editor.cc (variable_editor::maybe_resize_columns):
Don't reset size to default.

diff --git a/libgui/src/variable-editor.cc b/libgui/src/variable-editor.cc
--- a/libgui/src/variable-editor.cc
+++ b/libgui/src/variable-editor.cc
@@ -422,18 +422,16 @@ variable_editor::maybe_resize_columns (v
             {
               if (table->columnWidth (i) > mx)
                 mx = table->columnWidth (i);
             }
 
           table->horizontalHeader ()->setDefaultSectionSize (mx);
         }
     }
-  else
-    table->horizontalHeader ()->setDefaultSectionSize (m_default_width);
 }
 
 void
 variable_editor::contextmenu_requested (const QPoint& qpos)
 {
   QTableView *view = get_table_data (m_tab_widget).m_table;
   QModelIndex index = view->indexAt (qpos);
 
